makefile.shade

```diff
@@ -795,6 +805,19 @@ functions
 
             if(Directory.Exists(repo))
             {
+                if(UseGateBranch)
```

Comment:
`ci-build` has its own copy of the git clone script (on line 122). You need to update that bit rather than this.